src/components/ConfirmDialog.jsx

```diff
@@ -0,0 +1,113 @@
+import './ConfirmDialog.css';
```

Comment:
Nit from @ahsanatzapier consider having this at the bottom of the imports. Mostly to match general conventions of how imports generally happen.

This is correct and will work, but not how most folks do this.

Reply:
Thank you for this comment!